src/main/java/java/lang/String.java

```diff
@@ -3812,9 +3840,20 @@ public String toString() {
      * @diffblue.untested
      */
     public static String format(String format, Object... args) {
-        // DIFFBLUE MODEL LIBRARY This is treated internally in CBMC
-        return CProver.nondetWithoutNullForNotModelled();
+        // DIFFBLUE MODEL LIBRARY
```

Comment:
I suppose we need to constrain the size of `args` to be <= 10, otherwise we might generated incorrect tests using arguments that are ignored in cbmc, but not by the JVM.

Reply:
I made format return a nondet value in the case of more than 10 arguments, which is correct for the point of view of verification. I think it will be very rare to get a list of non-deterministic size here as arguments are normally passed explicitly and not through a list.

Reply:
What about when the input is nondet?

Reply:
OK, I now understand your comment about verification (although you allow false positives). I'll transfer my complaint to the "other" models-library :-).